app/controllers/transformation-controller.js

```diff
@@ -105,7 +104,14 @@ function getSurveyHash( req, res, next ) {
  */
 function _getFormDirectly( survey ) {
     return communicator.getXForm( survey )
-        .then( transformer.transform );
+        .then( communicator.getManifest )
+        .then( xform => {
+            const survey = Object.assign( {}, xform, {
+                media: utils.toMediaMap( xform.manifest ),
```

Comment:
I think the necessity is the other way around: `transform` needs to escape for consistency, as it's a public API in a standalone package.

If I remember correctly, this change was introduced before the change in transformer to escape all `media` keys and values, so it may not be strictly necessary here. But in my opinion it's probably better to also escape here for consistency/correctness.